### ISSUE TITLE
The requested URL /callback was not found on this server.

### ISSUE THREAD

**Issue by scaviator** [aaronpk/sample-oauth2-client#1]
Hello,

I ran the sample github.php app and get the following error:

![image](https://user-images.githubusercontent.com/28540780/42913559-8298fc1e-8aaa-11e8-9307-e9afb33db8ab.png)

Is there something we need to configure in the github.php file in order to account for the /callback

Environment:
Ubuntu 16.04 server with LAMP stack, configured with self-signed cert.

Thanks

**Comment by aaronpk** [comment]
Sorry about this! I realized the screenshot in the book is misleading. 

When you create the app on GitHub, you'll need to set the callback URL to `http://127.0.0.1/index.php` (if you're following the instructions from the book) or `http://127.0.0.1/github.php` (if you're using the code in this repo).

The screenshot in the book shows `https://example-app.com/callback` as the callback URL since that is the URL I used when testing the code.

If you are running the code on a remote server, then you just need to make sure that your callback URL in the GitHub app matches the URL that you are accessing the application from.
